torch_xla/csrc/tensor.h

```diff
@@ -51,10 +51,6 @@ class XLATensor {
 
   at::Tensor ToTensor();
 
-  // This API should be called instead of ToTensor() when the tensor is passed
```

Comment:
No please, leave this.

Reply:
We can always add it back.

Reply:
It's 3 lines of code and it's the mirror or ToTensor().
Makes no sense to remove.

Reply:
Ok.